eng/validation/build-arcadewithrepo.ps1

```diff
@@ -167,10 +178,14 @@ function Get-Github-RepoAuthUri($repoName)
     "https://${githubUser}:${githubPAT}@github.com/${githubOrg}/${repoName}"
 }
 
+function Get-AzDO-RepoAuthUri($repoName)
```

Comment:
Is this called from many places? If not I'd suggest to inline it.

Reply:
Inlined both of these :)